doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +94,59 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
 
 The Cloud Services and the Cloud Resources Brokers provide value-added services in addition to the fundamental capabilities like service and resource discovery.  These Brokers are critical for a multi-cloud environment to function and utilise cloud specific plugins to perform the necessary activities. These Brokers can, for example, provision and manage environments with resources and services for Machine Learning (ML) services, Augmented/Virtual Reality, or specific industries.
 
+<a name="8.2.4"></a>
+### 8.2.4 Multi-Cloud Interactions Model
+<a name="8.2.4.1"></a>
+### 8.2.4.1 Introduction
+To realise a federated cloud requires definition and agreement of set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
+
+As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Service Broker". The set of interactions that these interface points need to provide are defined by the "Multii-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
```

Comment:
```suggestion
As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Service Broker". The set of interactions that these interface points need to provide are defined by the "Multi-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
```
Typo Multii

Reply:
@ulikleber There is also another suggested change above - one of the duplicates "Cloud Service Broker" needs to be "Cloud Resources Broker".

Reply:
@pgoyal01 I saw it, but didn't know how to do it properly. Trying again ;-) by adding your change manually.

```suggestion
As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Resources Broker". The set of interactions that these interface points need to provide are defined by the "Multi-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
```

Reply:
@ulikleber Thanks

Reply:
@pgoyal01 & @ulikleber Thank you both, I believe I have corrected both errors.